release notes/v0.37.0.md

```diff
@@ -0,0 +1,73 @@
+k6 v0.37.0 is here! 🎉 Mainly it contains fixes and ongoing efforts with refactoring, but it's just because we are preparing some incredible things in upcoming releases!
+
+## New Features!
+
+### Added support for local files to --log-output ([#2285](https://github.com/grafana/k6/pull/2285))
+
+Added an option to output k6 logs to a file, on top of the supported stdout/stderr and loki instance.
+
+The feature speaks for itself with examples of usage:
+
+```sh
+k6 run --log-output file=./k6.log --logformat json ./examples/stages.js
+```
+
+with a defined minimal log level:
+
+```sh
+k6 run --log-output file=./k6.log,level=info --logformat json ./examples/stages.js
+```
+
+Thanks, @[alyakimenko](https://github.com/alyakimenko) for the contribution!
+
+**Docs**: [Using file output](https://k6.io/docs/using-k6/options/#file)
+
+### Added experimental basic event loops ([#882](https://github.com/grafana/k6/issues/882))
+
+We added basic event loop functionality ([#2228](https://github.com/grafana/k6/pull/2228) and [#2373](https://github.com/grafana/k6/pull/2373)). This is a first step and currently isn't used by anything, but is now available to extensions. In following releases this will be used by k6 and will mean that some APIs will start having callbacks or return Promises. But first it's likely that it will be used in some experimental extensions.
```

Comment:
Can we add a Javascript code example of what is possible now?

Reply:
We had one in the initial PR but later decided to remove it entirely since it's not ready for advertising https://github.com/grafana/k6/pull/2419#discussion_r822562236.

Reply:
> but is now available to extensions.

vs

>  it's not ready for advertising

Found some discrepancies in these sentences

Reply:
Good point :thinking:

Maybe we should change like:

```
but is now available as an experimental extension
``

@MStoykov wdyt?

Reply:
but it's not an extension - it is in the k6 core we will just be dropping it probably as soon as possible. We might have parts of it as extension, maybe even for the release - but not as part of k6.

Reply:
Out of curiosity, should we be wrapping these "experimental" features with command-line feature-flags? 

> dropping it probably as soon as possible

Just for my personal clarification, do you mean "dropping" as delivering? or discarding?

Reply:
> Out of curiosity, should we be wrapping these "experimental" features with command-line feature-flags?

That is an interesting idea but we have no such practice. So adding something like that for this would've likely meant that we wouldn't have shipped anything

> Just for my personal clarification, do you mean "dropping" as delivering? or discarding?

discarding

Reply:
What if I remove the "but is now available to extensions." ?

Reply:
https://github.com/MStoykov/xk6-events here is the basic extension that I pulled out, code by @na--

Reply:
```suggestion
We added basic event loop functionality ([#2228](https://github.com/grafana/k6/pull/2228) and [#2373](https://github.com/grafana/k6/pull/2373)). This is a first step and currently isn't used by anything, but is now available to extensions ([an example](https://github.com/MStoykov/xk6-events)). In following releases this will be used by k6 and will mean that some APIs will start having callbacks or return Promises. But first, it's likely that it will be used in some experimental extensions.
```

How about this?

Reply:
seems okay to me